### ISSUE TITLE
NTR: Zinc low in breast milk (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8361]
New term request Zinc low in breast milk (elthat_130818180553):   UMLS: 1842485

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000700
- label: Reduced breast milk zinc concentration
- parents: [Abnormal tissue metabolite concentration (HP:0032243)](https://hpo.jax.org/app/browse/term/HP:0032243)
- pmid: PMID:35230756
- def: The concentration of zink in breast milk is below the lower limit of normal.
- comment: Zinc is a normal component in human milk. One known cause is below-normal milk zinc levels due to maternal genetic mutations affecting mammary zinc transport proteins.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8361
- synonyms:
